src/SprykerEco/Yves/Payone/Handler/ExpressCheckout/QuoteHydrator.php

```diff
@@ -158,7 +170,7 @@ protected function hydrateQuoteWithCustomer(
             $quoteTransfer->setCustomer($customerTransfer);
             return $quoteTransfer;
         }
-
+        $customerTransfer->setIsGuest(true);
```

Comment:
isn't this set by default checkout process?

Reply:
No, it's not. I checked it. Without this flag customer created after placing order.